### ISSUE TITLE
[DevOps] Deal with secret scanning alerts.

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-team#80746]
## Description
There are three secret scanning alerts in one of our repositories that I should address.

See [thread](https://dsva.slack.com/archives/C02G6AB3ZRS/p1713204942379629)

## Tasks
- [x] [DevOps] Figure out what needs to be done to address these.
- [x] [DevOps] Address them.
- [x] [DevOps] Document or followup as needed.

## Acceptance Criteria

- [x] I've figured out what needs to be done here.
- [x] I've either done it, or 
- [ ] I've opened followup ticket(s) to address as needed.

**Comment by ndouglas** [comment]
All three were JWTs for dev environments and never contained any active credentials for production instances of any services. In addition, all expired almost two years ago, within minutes of the times at which they expired. I documented this in the individual alerts and closed them. I don't believe any followup is necessary.

**Comment by ndouglas** [comment]
@loripusey @anita-halici I did this while waiting on a blocker on #72263. I brought it in per Stephen and Adrian in this Slack thread: https://dsva.slack.com/archives/C02G6AB3ZRS/p1713204942379629.
